### ISSUE TITLE
New .Attributed won't work

### ISSUE THREAD

**Issue by bwk4u** [NonFactors/MVC6.Grid#86]
I was trying the example given in NonFactor webpage

`    .Attributed(new { data_type = "individuals" })
`

but it won't assign any attribute to the table, whereas if I remove the

`           .Css("table-hover")
`

it will remove the table-hover class from the table. Could I be missing something?

**Comment by Muchiachio** [comment]
Did you update `_Grid.cshtml` with the newest one?

**Comment by bwk4u** [comment]
Thanks! Now I did and the attributes are working as expected. Nonetheless, .Sorteable() stopped working, and it only works for me when using version <= 1.4.0, not with the new ones. Any heads up for that?

By the way, i'm on core .NET, and for installing new cshtml, css and js I had to install (on another solution) the non-core nugget , and copy all the Content files to my CORE solution, since the core nugget doesn't create the Content folder.
